mobile/CmpE451_App/src/screen/CreatePost.js

```diff
@@ -212,125 +108,54 @@ export default function CreatePost({route}) {
           maxHeight: '50%',
           alignItems: 'flex-start',
         }}>
-        {index ? (
-          <View>
+        {textInputs.map((input, key) => (
+          <View style={styles.inputContainer}>
             <TextInput
-              placeholder="Add a title"
-              placeholderTextColor="black"
-              style={{alignSelf: 'flex-start', marginLeft: 10, fontSize: 20}}
+              placeholder={input.key}
```

Comment:
```suggestion
              placeholder={input.key + '(text)'}
```
If we add the type of field in brackets, the user can understand what is requested.